homeassistant/components/watson_tts/tts.py

```diff
@@ -22,39 +22,47 @@
 
 # List from https://tinyurl.com/watson-tts-docs
```

Comment:
Please remove the unrelated files

Reply:
Hmmm, must have rebased wrong somewhere, apologies